applications/StructuralMechanicsApplication/custom_response_functions/adjoint_conditions/adjoint_semi_analytic_point_load_condition.cpp

```diff
@@ -181,6 +179,9 @@ namespace Kratos
         KRATOS_CATCH( "" )
     }
 
+    // TODO find out what to do with KRATOS_API
+    template class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) AdjointSemiAnalyticPointLoadCondition<PointLoadCondition>;
```

Comment:
I think here you need the KRATOS_API_EXTERN right @jcotela?

Reply:
The extern thing is mostly for registering in kratos components (to make sure we don't end up with different instances of a member variable in different libraries). In this context, I would not even put the KRATOS_API, but not 100% sure on that.

Reply:
According to https://github.com/KratosMultiphysics/Kratos/commit/1e3e657c6f226754607ae2b4952fd6293c1b5ec4
KRATOS_API is necessary in windows.

Reply:
I also had to do this for the TL element. It looks correct to me.